keps/sig-storage/3751-volume-provisioned-io/README.md

```diff
@@ -0,0 +1,1338 @@
+
+For Beta and GA, add links to added tests together with links to k8s-triage for those tests:
+https://storage.googleapis.com/k8s-triage/index.html
+
+We expect no non-infra related flakes in the last month as a GA graduation criteria.
+-->
+
+### Graduation Criteria
+
+#### Alpha
+
+- Feature in 1.28 behind the VolumeAttributesClass feature gate which is set to a default of false
+- Add basic unit tests
+
+#### Alpha -> Beta
+- Unit tests and e2e tests outlined in design proposal implemented
+
+#### Beta
+
+- Beta in 1.29: Since this feature is an expansion of the existing capacity usage flow, we are going to move this to beta with enhanced e2e and test coverage 
+
+#### GA
+
+- GA in 1.30, an issue board will be created to track issues, and all major issues should be fixed before GA.
+- No users complaining about the new behavior
+
+### Upgrade / Downgrade Strategy
+
+No change in cluster upgrade / downgrade process.
+
+### Version Skew Strategy
+
+This feature is implemented only in the API server and KCM and controlled by
+`VolumeProvisionedIO` feature gate. Following cases may happen:
+
+| API server | external-resizer | Behavior                                                                                                                         |
+|------------|-----|----------------------------------------------------------------------------------------------------------------------------------|
+| off | off | Existing Kubernetes behavior.                                                                                                    |
+| on | off| Existing Kubernetes behavior, API server gets requests of iops/throughput but does not act on it
+| off | on | external-resizer should not get any event to update iops/throughput, the current resize flow stays the same |
+| on | on | New behavior.                                                                                                                    |
+
+## Production Readiness Review Questionnaire
+
+### Feature Enablement and Rollback
+
+###### How can this feature be enabled / disabled in a live cluster?
+
+- [ ] Feature gate (also fill in values in `kep.yaml`)
+    - Feature gate name: VolumeProvisionedIO
+    - Components depending on the feature gate: kube-apiserver, 
+      external-resizer
+- [ ] Other
+    - Describe the mechanism:
+
+###### Does enabling the feature change any default behavior?
+
+No. 
+
+###### Can the feature be disabled once it has been enabled (i.e. can we roll back the enablement)?
+
+Yes. It should be disabled in a reverse order of enabling the feature - 
+first disable the feature in external-resizer then in API server. See "Version Skew 
+Strategy" section above for more details.
+
+
+###### What happens if we reenable the feature if it was previously rolled back?
+
+No issues are expected. The case is exactly the same as when the feature is 
+enabled for the first time.
```

Comment:
Do `VolumeAttributesClassName` values already written to storage (from before the feature was rolled back) become active again?

Reply:
It does not get active again because the PVC's VolumeAttributesClassName does not change.

Reply:
Would you update the KEP to summarize that situation here?  My goal is to understand what happens when the feature is enabled, `VolumeAttributesClassName` is set, and then the feature is disabled and then reenabled again.